src/AddTodoForm.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 
-const AddTodoForm = () => {
-    return(
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        let todoTitle = event.target.value;
+        console.log(todoTitle);
+        onAddTodo(todoTitle);
```

Comment:
There's an error in the browser when you try to open the app and enter a value in the input
`Uncaught ReferenceError: onAddTodo is not defined`